FF1Lib/Flags.cs

```diff
@@ -730,6 +743,11 @@ public static Flags DecodeFlagsText(string text)
 				ClampBossHPScaling = GetTriState(ref sum),
 				SeparateEnemyHPScaling = GetTriState(ref sum),
 				SeparateBossHPScaling = GetTriState(ref sum),
+				PoolSize = (PoolSize)GetNumeric(ref sum, Enum.GetValues(typeof(PoolSize)).Cast<int>().Max() + 1),
+				EnablePoolParty = GetBoolean(ref sum),
+				IncludePromClasses = GetTriState(ref sum),
+				EnableRandomPromotions = GetBoolean(ref sum),
+				IncludeBaseClasses = GetTriState(ref sum),

```

Comment:
These need to be in the same place in the encoding and decoding order, notice how the spoilers flag is at the end of the encoding and its at the end of the decoding